low-level/json_msgs/messages/sensors/iem_data.py

```diff
@@ -76,7 +77,6 @@ def __init__(self,
                         "component": info.get("component_id"),
                         "module": info.get("module_id"),
                         "event": info.get("event_id"),
-                        "description": info.get("description"),

```

Comment:
confirm about IEM, what field being used again, as this has description already

Reply:
Discussed with soniya from CSM team , with latest change for all alerts including IEM , they will  use "description" key from info to display the Description on CSM UI.

Reply:
Ok